### PR TITLE
Designing page

### PR DESCRIPTION
**Pull request by imatif01**
- change header background color.
- change Logo.
- change background color of search bar and profile drop down.
- Add section in user detail page.
- add Progress bars in user detail page.
- Fix input fields issues of user detail page.
- Add title of user detail page.
- Change Main image of User detail page.
- Remove filter section from top of the user Management page.
- Change Country section in table of user Management page.
- Remove icons from field Action of user Management page.
- Add progress bars in user Management page.
- Change user details in user Management page.
- Add chat button in the of user details in user Management page.

Please Review following changes.

### CODE REVIEW

**Review by mosmartin — COMMENTED** [review]
Hi @imatif01, is the `react-testing-tutorial` relevant to the project?

File: src/Components/Common/ProfileDropdown.js [hunk 1]
```diff
@@ -90,27 +90,14 @@ const ProfileDropdown = () => {
...
-              to={process.env.PUBLIC_URL + "/pages-profile"}
-              className="dropdown-item"
-            >
-              <i className="mdi mdi-wallet text-muted fs-16 align-middle me-1"></i>{" "}
-              <span className="align-middle">
-                Balance : <b>$5971.67</b>
-              </span>
-            </Link>
-          </DropdownItem>
+          {/* <DropdownItem className='p-0'>
```
Comment (mosmartin) [inline src/Components/Common/ProfileDropdown.js:94]: Please clean up the commented code.

File: src/Layouts/Header.js [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { Link } from "react-router-dom";
 import { Dropdown, DropdownMenu, DropdownToggle, Form } from "reactstrap";
 
 //import images
-import logoSm from "../assets/images/logo-sm.png";
-import logoDark from "../assets/images/logo-dark.png";
-import logoLight from "../assets/images/logo-light.png";
+// import logoSm from "../assets/images/logo-sm.png";
```
Comment (mosmartin) [inline src/Layouts/Header.js:6]: As above, please remove all commented code as it's not necessary.

File: src/pages/Authentication/ForgetPassword.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ const ForgetPasswordPage = (props) => {
     forgetSuccessMsg: state.ForgetPassword.forgetSuccessMsg,
   }));
 
-  document.title = "Reset Password | Velzon - React Admin & Dashboard Template";
+  document.title = "Reset Password |GreenMe";
```
Comment (mosmartin) [inline src/pages/Authentication/ForgetPassword.js:57]: Kindly maintain the space before and after the pipe `|` so it's consistent.

```jsx
document.title = "Reset Password | GreenMe";
```

File: src/pages/Authentication/Register.js [hunk 4]
```diff
@@ -82,7 +82,7 @@ const Register = () => {
     }, 3000);
   }, [dispatch, success, error, history]);
 
-  document.title = "Basic SignUp | Velzon - React Admin & Dashboard Template";
+  document.title = "Basic SignUp |GreenMe";
```
Comment (mosmartin) [inline src/pages/Authentication/Register.js:85]: As above, please maintain the space before and after the pipe `|` for consistency.

File: src/pages/Authentication/user-profile.js [hunk 5]
```diff
@@ -76,7 +76,7 @@ const UserProfile = () => {
     },
   });
 
-  document.title = "Profile | Velzon - React Admin & Dashboard Template";
+  document.title = "Profile |GreenMe";
```
Comment (mosmartin) [inline src/pages/Authentication/user-profile.js:79]: Same as above and all other instances.

File: src/Components/Common/ProfileDropdown.js (hunk 1 above)
Comment (imatif01) [reply src/Components/Common/ProfileDropdown.js:94]: This commented code can be used in future, that's why I haven't deleted yet.

File: src/Layouts/Header.js (hunk 2 above)
Comment (imatif01) [reply src/Layouts/Header.js:6]: I remove it

File: src/pages/Authentication/ForgetPassword.js (hunk 3 above)
Comment (imatif01) [reply src/pages/Authentication/ForgetPassword.js:57]: fix it

File: src/pages/Authentication/Register.js (hunk 4 above)
Comment (imatif01) [reply src/pages/Authentication/Register.js:85]: Fix it

File: src/pages/Authentication/user-profile.js (hunk 5 above)
Comment (imatif01) [reply src/pages/Authentication/user-profile.js:79]: Fix it

File: src/Components/Common/ProfileDropdown.js (hunk 1 above)
Comment (mosmartin) [reply src/Components/Common/ProfileDropdown.js:94]: For anything code that is to be deployed to staging and prod, no unnecessary comments are required. You can have them in your feature branch but we'll need them cleaned before we can merge them.

**Review by mosmartin — APPROVED** [review]
This PR has a lot of technical debt that needs to be looked into.

**Review by Mbocha — APPROVED** [review]

**Pull request merged**